bedrock/exp/templates/exp/firefox/new/download.html

```diff
@@ -27,9 +27,18 @@
 {#- This will appear as <meta property="og:description"> which can be used for social share -#}
 {% block page_og_desc %}Faster page loading, less memory usage and packed with features, the new Firefox is here.{% endblock %}
 
+{% block page_css %}
+  {% if v == 'b' %}
+  <style>
```

Comment:
We should really avoid inline CSS because CSP, but I don't think we're blocking it and this is a temporary experiment, so 👍